website/docs/r/cluster.html.markdown

```diff
@@ -96,6 +96,7 @@ resource "rancher2_cluster" "foo-custom" {
       "prometheus.resources.core.requests.memory" = "750Mi"
       "prometheus.retention" = "12h"
     }
+    version = "0.1.0"
```

Comment:
can we use a more modern version, if people copy and paste this seems to old.

Reply:
rancher-monitoring `v0.1.0` is the latest versions by the moment, https://github.com/rancher/system-charts/tree/dev-v2.5/charts/rancher-monitoring

Reply:
Oops, you are right, we went from 0.0.8 or something :)